rules/opentitan_test.bzl

```diff
@@ -249,7 +249,7 @@ def opentitan_functest(
         manifest = "//sw/device/silicon_creator/rom_ext:manifest_standard",
         slot = "silicon_creator_a",
         test_harness = "@//sw/host/opentitantool",
-        key = "test_key_0",
+        key = "prod_key_0",
```

Comment:
Can we add a comment making it abundantly clear that this is not **the** prod key, but a testing key that sits in the place of the prod key for FPGA and simulation based testing?

Reply:
Good point. Sure, I'll actually add the `fake_` prefix to the keys of the `DEFAULT_SIGNING_KEYS` dict in `rules/opentitan.bzl` so that we don't have to add a comment every time we specify a key.

Reply:
If you add such a prefix, can you be sure to update the DV testbench here: https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/dv/env/chip_env_cfg.sv;drc=b2ef5fac6b1c248181cb9619095bcca099f7b4d2;l=393

Otherwise, the ROM E2E test in DV nightlies will fail. They are not blocking in CI as they take a long time to run, though I might add one.

Reply:
Haha, I was about to ping you for a review :)

Would you mind taking a look at the last commit?